src/HealthGPS/static_linear_model.cpp

```diff
@@ -21,10 +21,9 @@ StaticLinearModel::StaticLinearModel(
     if (!risk_factor_cholesky_.allFinite()) {
         throw core::HgpsException("Risk factor Cholesky matrix contains non-finite values");
     }
-    // TODO: Uncomment when fully implemented
-    // if (weight_quantiles_.empty()) {
-    //    throw core::HgpsException("Weigth quantiles dictionary is empty");
-    //}
+    if (weight_quantiles_.empty()) {
+        throw core::HgpsException("Weight quantiles dictionary is empty");
+    }
```

Comment:
I'd add the same check to `*ModelDefinition` below.

Reply:
Done!